modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -392,72 +392,6 @@ public ObjectDefinition addSystemObjectDefinition(
 			true, titleObjectFieldName, version, status, objectFields);
 	}
 
-	@Override
-	public void bindObjectDefinitions(long[] objectRelationshipIds)
-		throws PortalException {
-
-		ObjectRelationship rootObjectRelationship =
-			_objectRelationshipLocalService.getObjectRelationship(
-				objectRelationshipIds[objectRelationshipIds.length - 1]);
-
-		long rootObjectDefinitionId =
-			rootObjectRelationship.getObjectDefinitionId1();
-
-		ObjectDefinition rootObjectDefinition =
-			objectDefinitionLocalService.getObjectDefinition(
-				rootObjectDefinitionId);
-
-		if (rootObjectDefinition.getRootObjectDefinitionId() == 0) {
-			objectDefinitionLocalService.updateRootObjectDefinitionId(
-				rootObjectDefinitionId, rootObjectDefinitionId);
-		}
-
-		for (long objectRelationshipId : objectRelationshipIds) {
-			ObjectRelationship objectRelationship =
-				_objectRelationshipLocalService.getObjectRelationship(
-					objectRelationshipId);
-
-			if (objectRelationship.isEdge()) {
-				continue;
-			}
-
-			objectRelationship.setDeletionType(
-				ObjectRelationshipConstants.DELETION_TYPE_CASCADE);
-			objectRelationship.setEdge(true);
-
-			objectRelationship =
-				_objectRelationshipLocalService.updateObjectRelationship(
-					objectRelationship);
-
-			ObjectDefinition objectDefinition1 =
-				objectDefinitionLocalService.getObjectDefinition(
-					objectRelationship.getObjectDefinitionId1());
-
-			if (objectDefinition1.getRootObjectDefinitionId() == 0) {
-				objectDefinition1 =
-					objectDefinitionLocalService.updateRootObjectDefinitionId(
-						objectDefinition1.getObjectDefinitionId(),
-						rootObjectDefinitionId);
-			}
-
-			updatePortlet(objectDefinition1.getObjectDefinitionId());
-
-			ObjectDefinition objectDefinition2 =
-				objectDefinitionLocalService.getObjectDefinition(
-					objectRelationship.getObjectDefinitionId2());
-
-			objectDefinition2 =
-				objectDefinitionLocalService.updateRootObjectDefinitionId(
-					objectDefinition2.getObjectDefinitionId(),
-					rootObjectDefinitionId);
-
-			_objectFieldLocalService.updateRequired(
-				objectRelationship.getObjectFieldId2(), true);
-
-			updatePortlet(objectDefinition2.getObjectDefinitionId());
-		}
-	}
```

Comment:
Hey @victorKammerer whenever a method is removed or created in *LocalServiceImpl.java files with the annotation `@Override` it should be put in one commit and then we have to run `gradlew buildService` in the service module so the files that are autogenerated will be updated, those changes should be put in another commit so we can easily identify which was autogenerated and which was not, also it will be necessary for backport purpose, so:

1. create one commit with the change removing the method of ObjectDefinitionLocalServiceImpl, this one can have the same message you have defined.
2. run `gradlew buildService` in the service module and put all changes generated by this command in another commit with the message `LPD-41332 Build Service`

if you have any questions feel free to reach me (messages might be delayed), or talk to @pedro-oliveira446 or @devpedrotavares 

Thank you!